common/internal/requestlogger_test.go

```diff
@@ -43,22 +39,20 @@ func TestNopLogger_FlushLog(t *testing.T) {
 	require.Empty(t, hook.Entries)
 
 	// and should be no-ops
-	rw := httptest.NewRecorder()
-	require.Equal(t, rw, l.ResponseWriter(rw))
+	//rw := httptest.NewRecorder()
+	//require.Equal(t, rw, l.ResponseWriter(rw))
```

Comment:
Commented out code. Are these checks necessary?

Reply:
This test `TestNopLogger_FlushLog` is being removed, as the logrus level is not valid, test `TestLogger_FlushLog` will test the hook entry.